automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/DataUseAwareSearchSpec.scala

```diff
@@ -52,6 +58,53 @@ class DataUseAwareSearchSpec extends FreeSpec with WebBrowserSpec with UserFixtu
         Label("commercial-tag").isVisible shouldBe false // We didn't select this one
       }
     }
+
+    "The ontology autocomplete exists and works" in withWebDriver { implicit driver =>
+      val user = UserPool.chooseAnyUser
+
+      withSignIn(user) { _ =>
+        val page = new DataLibraryPage().open
+
+        page.openResearchPurposeModal()
+
+        page.selectRPCheckbox("disease-focused-research")
+
+        // Disease 1
+        SearchField("ontology-autosuggest").setText("fatal")
+
+        val ffiSuggestionId = "suggestion-http://purl.obolibrary.org/obo/DOID_0050433"  // fatal familial insomnia
```

Comment:
From a maintenance perspective, I worry about proliferating DOID terms around the various codebases. DOID terms are subject to deprecation. So, not for this PR, but in general, maybe we should think about a way to centralize our use of terms. But even that might be more effort than it's worth to just update tests when terms do change.

Reply:
💯 This is killing me, I would ideally test the `ResearchPurposeSection` React component in isolation with mock data, but we just don't have the infrastructure to do that right now